frontend/src/app/shared/components/storages/openproject-storages.module.ts

```diff
@@ -92,4 +92,5 @@ import {
     StorageInformationService,
   ],
 })
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
 export class OpenprojectStoragesModule {}
```

Comment:
The TypeScript decorator needs to directly above the class definition.

You can put the ESLint directive on the same line:

```suggestion
export class OpenprojectStoragesModule {} // eslint-disable-line @typescript-eslint/no-extraneous-class
```

Reply:
I don't get it yet, what is the benefit from using `eslint-disable-line` over `eslint-disable-next-line`?

Reply:
> The TypeScript decorator (`@NgModule`) needs to directly above the class definition.